packages/hardware-trezor/src/transformers/tx.ts

```diff
@@ -1,50 +1,38 @@
 import * as Trezor from '@trezor/connect';
 import { Cardano } from '@cardano-sdk/core';
+import { Transformer, transformObj } from '@cardano-sdk/util';
 import { TrezorTxTransformerContext } from '../types';
 import { mapAdditionalWitnessRequests } from './additionalWitnessRequests';
 import { mapAuxiliaryData, mapCerts, mapRequiredSigners, mapTxIns, mapTxOuts, mapWithdrawals, toTxOut } from './';
 import { mapTokenMap } from './assets';
 
-/**
- * Temporary transformer function that returns a partial
- * trezor.CardanoSignTransaction object which can be merged
- * into the extisting implementation (later we should refactor
- * this function to the Transformer interface like in the
- * hardware-ledger package)
- */
-const trezorTxTransformer = (
-  body: Cardano.TxBody,
-  context: TrezorTxTransformerContext
-): Omit<Trezor.CardanoSignTransaction, 'signingMode'> => {
-  const inputs = mapTxIns(body.inputs, context);
-  return {
-    additionalWitnessRequests: mapAdditionalWitnessRequests(inputs, context),
-    auxiliaryData: body.auxiliaryDataHash ? mapAuxiliaryData(body.auxiliaryDataHash) : undefined,
-    certificates: mapCerts(body.certificates ?? [], context),
-    collateralInputs: body.collaterals ? mapTxIns(body.collaterals, context) : undefined,
-    collateralReturn: body.collateralReturn ? toTxOut(body.collateralReturn, context) : undefined,
-    fee: body.fee.toString(),
-    inputs,
-    mint: mapTokenMap(body.mint, true),
-    networkId: context.chainId.networkId,
-    outputs: mapTxOuts(body.outputs, context),
-    protocolMagic: context.chainId.networkMagic,
-    referenceInputs: body.referenceInputs ? mapTxIns(body.referenceInputs, context) : undefined,
-    requiredSigners: body.requiredExtraSignatures
-      ? mapRequiredSigners(body.requiredExtraSignatures, context)
-      : undefined,
-    totalCollateral: body.totalCollateral ? body.totalCollateral.toString() : undefined,
-    ttl: body.validityInterval?.invalidHereafter?.toString(),
-    validityIntervalStart: body.validityInterval?.invalidBefore?.toString(),
-    withdrawals: mapWithdrawals(body.withdrawals ?? [], context)
-  };
+export const trezorTxTransformer: Transformer<
+  Cardano.TxBody,
+  Omit<Trezor.CardanoSignTransaction, 'signingMode' | 'derivationType' | 'includeNetworkId'>,
```

Comment:
Are those properties never needed? :thinking:

Reply:
`derivationType` and `includeNetworkId` are never used. 
SigningMode is determined after data mapping.